README_AWS.md

```diff
@@ -1,3 +1,4 @@
+:warning: **WARNING** :warning: This feature is community supported and may contain bugs. Visit [docs.openshift.com](https://docs.openshift.com) for [OpenShift Enterprise](https://docs.openshift.com/enterprise/latest/install_config/install/index.html) or [OpenShift Origin](https://docs.openshift.org/latest/install_config/install/index.html) supported installation docs.
```

Comment:
Swap `may contain bugs` with `has not been tested by Red Hat` ? Just trying to avoid the implication that other bits are bug free.